lib/pushmi_pullyu/cli.rb

```diff
@@ -181,14 +182,20 @@ def rotate_logs
   end
 
   def run_preservation_cycle
-    item = queue.wait_next_item
-    return unless item.present?
+    entity_json = JSON.parse(queue.wait_next_item)
+    entity = {
+      type: entity_json['type'],
+      uuid: entity_json['uuid']
+    }
+    return unless entity.present?
 
-    # add additional information about the error context to errors that occur while processing this item.
-    Rollbar.scoped(noid: item) do
+    # add additional information about the error context to errors that occur
+    # while processing this item.
+    Rollbar.scoped(entity_uuid: entity[:uuid]) do
       begin
-        # Download AIP from Fedora, bag and tar AIP directory and cleanup after block code
-        PushmiPullyu::AIP.create(item) do |aip_filename, aip_directory|
+        # Download AIP from Jupiter, bag and tar AIP directory and cleanup after
+        # block code
```

Comment:
We use 120 line length on all of our projects which rubocop enforces, no need to split these comments to two lines ¯\_(ツ)_/¯

Reply:
I am used to 80 chars max length, I did not see that the configuration was changed to 120. I can definitely stick to the 120 since that is what we are using.

Reply:
Should we add this to our .editorconfig as well rather than having rubocop catch it after?

Reply:
I don't believe editorconfig supports max line length (quick google search returns info about researching if it's possible, nothing saying it is available), otherwise for sure we should add it as well.

Reply:
I saw it [here](https://devhints.io/editorconfig) but I guess that's not definitive.  I can see now that there is a lot [of controversy](https://github.com/editorconfig/editorconfig/issues/387) over whether to include or exclude a rule like this. Disregard my suggestion about .editorconfig